package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "filmcards",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
```

Comment:
@testing-library/user-event should be installed as a development dependency.
https://stackoverflow.com/a/22004559/11261449